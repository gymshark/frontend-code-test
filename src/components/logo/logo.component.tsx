import { GymsharkIcon } from './logo.svg';

import * as S from './logo.styles';

export const Logo = (): JSX.Element => (
    <S.LogoWrapper>
        <GymsharkIcon />
    </S.LogoWrapper>
);
