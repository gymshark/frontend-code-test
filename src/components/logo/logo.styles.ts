import styled from 'styled-components';

export const LogoWrapper = styled.div`
    display: flex;
    height: 100vh;

    svg {
        width: 10rem;
        margin: auto;
    }
`;
