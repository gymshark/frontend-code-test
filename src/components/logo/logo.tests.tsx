import '@testing-library/jest-dom';
import { render } from '@testing-library/react';

import { Logo } from './logo.component';

test('Logo Snapshot', () => {
    const { asFragment } = render(<Logo />);

    expect(asFragment()).toMatchSnapshot();
});
