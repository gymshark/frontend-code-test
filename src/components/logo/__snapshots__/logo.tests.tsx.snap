// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdnxRM bGiBra"
  >
    <svg
      version="1.1"
      viewBox="0 0 30.4 23.2"
      x="0"
      xml:space="preserve"
      xmlns="http://www.w3.org/2000/svg"
      y="0"
    >
      <path
        d="M0 0h30.4s-14 15.5-21 23.2c.3-.6.5-1.1.9-1.6-.3 0-.7-1.3-1-1.9l-1.2.6c-.2.2-.6.1-.8-.1 0-.1-.1-.1-.1-.2-.2-.4-.3-.8-.5-1.3-.5.3-1 .5-1.5.7L.3 17.3h3.5l1.6 1.3c.4-.4.7-1 1-1.3h.9c.3.3.6.9 1 1.3.4-.4.8-.9 1.2-1.2h.6c.3.3.7.8 1 1.1h.1s.6-.8.9-1.1h.7s9-13.5 9-13.6C14.6 2.5 0 0 0 0"
        fill="#010101"
      />
      <path
        d="M8.9 8L3.9 5.2 12.8 7.3z"
        fill="#010101"
      />
    </svg>
  </div>
</DocumentFragment>
`;
